### ISSUE TITLE
Weird responses from A9G

### ISSUE THREAD

**Issue by inpoison** [IOT-MCU/ESP-12S-A9-A9G-GPRS-Node-v1.0#5]
Hi,
I wanted to the test the board with example "Get GPS Location" mentioned in the wiki.
Unfortunately, the responses from the A9G seem to be weird (and not successful):
![image](https://user-images.githubusercontent.com/30077252/81320500-93a5dd80-9091-11ea-83a8-4c4625e70ca4.png)
Do you know what's the problem?

Thanks in advance,
Tobi

**Comment by IOT-MCU** [comment]
Hi，
Do you power the board via a 3.7V lipo battery? Battery must need.

**Comment by inpoison** [comment]
Yes, a 3,7V (600mAh) battery is plugged in and the orange LED is on. The standard baud rate to communicate with the A9G is 115200, right?

**Comment by IOT-MCU** [comment]
yes, the default baud rate is 115200. 
It seems that the A9G can not power on by the ESP-12S. Please down load the ESP-12S_A9G_TEST.ino to test the board again. Before shipping, i had test all the board. 
If it can not working, please sent me more detail, Thanks.

**Comment by inpoison** [comment]
This is the output with the mentioned ESP-12S_A9G_TEST.ino:
![image](https://user-images.githubusercontent.com/30077252/81719760-c2e79080-947d-11ea-9513-166a8110058c.png)

**Comment by IOT-MCU** [comment]
Hi， The A9G not power on. Please show me the hardware connection.

**Comment by svenb** [comment]
Hi, i also get not gps connect, also connected a battery but I get this output from the test.ino

21:46:42.288 -> analogRead:2
21:46:49.308 -> analogRead:1 

any idea?
